test/init-update-dot-files/expected-stdout

```diff
@@ -1,3 +1,4 @@
+Loaded env file .env.local
```

Comment:
Asi by som dal cestu do uvodzoviek, ked to tak mame inde, ako je to vidiet v tomto teste:
`Loaded env file ".env.local"`

Reply:
👍 upravím